ibm/resource_ibm_pi_network_port.go

```diff
@@ -44,14 +45,9 @@ func resourceIBMPINetworkPort() *schema.Resource {
 				Optional: true,
 			},
 
-			/*
-			 "description": "",
-			    "ipAddress": "172.31.96.93",
-			    "macAddress": "fa:16:3e:30:b9:64",
-			    "portID": "6c9d0e42-73f3-492f-840d-3d4a7a573014",
-			*/
+			//Computed Attributes
 
-			helpers.PINetworkPortIPAddress: {
+			"ipaddress": {
```

Comment:
The required and option fields for all power resources are having this naming convnetion helpers.PINetworkPortIPAddress (pi_XXXX) I think please follow same here also